firebase-app-distribution/src/test/java/com/google/firebase/app/distribution/FirebaseAppDistributionTest.java

```diff
@@ -374,14 +393,6 @@ public void updateToNewRelease_whenCheckForUpdateFails_updateAppNotCalled() {
     assertEquals(FirebaseAppDistributionException.Status.NETWORK_FAILURE, e.getErrorCode());
   }
 
-  @Test
-  public void updateToNewRelease_callsSignInTester() {
```

Comment:
Just checking: did you remove this test because it's only verifying a method call?

Reply:
Ah yes, I meant to leave a note here. I felt this test was testing implementation, not behavior, so I removed it